engine/src/test/java/org/kigalisim/validate/LiveTests.java

```diff
@@ -405,45 +405,152 @@ public void testBasicReplace() throws IOException {
     // Check Sub B in year 1
     EngineResult resultSubB1 = findResult(results, "Test", "Sub B", 1);
     assertNotNull(resultSubB1, "Should have result for Test/Sub B in year 1");
-    
+
     System.out.println("Sub B Year 1 - Manufacture: " + resultSubB1.getManufacture().getValue() + " " + resultSubB1.getManufacture().getUnits());
     System.out.println("Sub B Year 1 - Import: " + resultSubB1.getImport().getValue() + " " + resultSubB1.getImport().getUnits());
     System.out.println("Sub B Year 1 - Recycle: " + resultSubB1.getRecycle().getValue() + " " + resultSubB1.getRecycle().getUnits());
     System.out.println("Sub B Year 1 - GHG: " + resultSubB1.getGhgConsumption().getValue() + " " + resultSubB1.getGhgConsumption().getUnits());
 
     // Move to year 5 and check results (start of replacement)
     for (int year = 2; year <= 5; year++) {
-      engine.incrementYear();
+      engine.incrementYear();  // This could be a problem.
```

Comment:
This isn't actually running calculations. Please transition `LiveTests` to use `KigaliSimFacade`.